lib/ballot.go

```diff
@@ -100,6 +99,22 @@ func (b Ballot) String() string {
 	return string(encoded)
 }
 
+func (b Ballot) IsFitInVoting() (ret bool) {
+	switch b.State() {
+	case sebakcommon.BallotStateSIGN:
+	case sebakcommon.BallotStateACCEPT:
+	case sebakcommon.BallotStateALLCONFIRM:
```

Comment:
`sebakcommon.BallotStateALLCONFIRM` is not for voting, it must be return `false` and in the expiring process it should be removed from box for later.